app/src/main/java/org/simple/clinic/newentry/ButtonState.kt

```diff
@@ -0,0 +1,6 @@
+package org.simple.clinic.newentry
+
+enum class ButtonState {
+  CLICKED,
```

Comment:
Can we rename the button states to be more domain-specific like `SAVING`, `SAVED`? Currently, these are named to be more screen-specific and we usually avoid that particular kind of naming when placing inside the Mobius model.